.ci/table_of_content.py

```diff
@@ -0,0 +1,106 @@
+import json
+import pathlib
+import argparse
+import re
+
+TABLE_OF_CONTENT = "Table of content:"
```

Comment:
Table of content could not include a semicolon sign. Expression could be `#+\s+Table of content:?` to avoid matches in non-header cells.

Reply:
thank you, changed